docs/architecture/databaseUtil.md

```diff
@@ -0,0 +1,51 @@
+# DatabaseUtil
```

Comment:
The missing `HasGameFile` and `Delete` diagrams were added in [this commit](https://github.com/normal-devs/klicker-knight/pull/23/commits/49e6cbc7a5d6e7dafe96b48b30e11a69d4311167)